assets/js/pages/BattleDetails.js

```diff
@@ -213,11 +214,18 @@ BattleDetails.propTypes = {
       pk: PropTypes.string,
     }),
   }).isRequired,
+  loadBattle: PropTypes.func.isRequired,
+  battle: PropTypes.any.isRequired,  // eslint-disable-line
```

Comment:
Why is this `any`?

Reply:
When I put `PropTypes.array` it gives me an error saying that I provided an `object`. When I put `PropTypes.object` the error is because I provided an `array`. It's weird.